src/components/RankingTable.js

```diff
@@ -10,25 +12,69 @@ const styles = {
 		width: '100%',
 		maxWidth: contentMaxWidth,
 		flexDirection: 'column',
-		backgroundColor: paper,
-		boxShadow: [[1, 1, 2, 2, shadow]],
-		border: [[1, 'solid', black]],
+		backgroundColor: yellow,
 		height: 'fit-content',
-		marginBottom: 16,
+		marginBottom: 50,
 	},
 	headerRow: {
-		textTransform: 'uppercase',
 		fontWeight: 'bold',
+		textAlign: 'left',
+		'& div': {
+			border: 0,
+		},
+	},
+	podiumRow: {
+		backgroundColor: black,
+		color: '#fff',
+		border: 'none',
+	},
+	hideLosers: {
+		display: 'none',
+	},
+	hideWinners: {
+		display: 'none',
+	},
+	starDivider: {
 		textAlign: 'center',
+		...robotoBold,
+		color: black,
+		lineHeight: '90%',
```

Comment:
Don't use percentages for things like this. [From Mozilla on line-height: Percentage values may produce unexpected results...](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height)